src/analysis/CMakeLists.txt

```diff
@@ -172,7 +172,7 @@ INCLUDE_DIRECTORIES(SYSTEM
   ${SQLITE3_INCLUDE_DIR}
 )
 
-REMOVE_DEFINITIONS("-DANALYSIS_EXPORT=${DLLIMPORT}")
+ADD_DEFINITIONS("\"-UANALYSIS_EXPORT\"")
```

Comment:
I don't think -U should be used with ADD_DEFINITIONS. See http://cmake.3232098.n2.nabble.com/undef-a-symbol-td7580326.html